biodatacatalyst-ui/src/main/webapp/picsureui/overrides/footer.js

```diff
@@ -30,6 +27,24 @@ define(["handlebars", "text!overrides/footer.hbs"], function(HBS, template){
 			}
 			$('title').html(title);
 			this.$el.html(HBS.compile(template)());
+
+			if (session.isValid()) {
+				$(document).ready(function () {
+					$('a[target="_blank"]').on('click', function (event) {
+						event.preventDefault();
+						console.log(event.target.href);
+
+						const linkView = new linkRedirect({href: event.target.href});
+						modal.displayModal(
+							linkView
+							, "Leaving BDC-PIC-SURE"
+							, () => {
+							}
+							, {handleTabs: true, width: "450px"}
+						);
+					});
+				});
+			}
```

Comment:
You could use https://github.com/hms-dbmi/biodatacatalyst-pic-sure/blob/master/biodatacatalyst-ui/src/main/webapp/picsureui/common/pic-sure-dialog-view.js istead of using a new view.

Reply:
Example: https://github.com/hms-dbmi/biodatacatalyst-pic-sure/blob/6cc894ddd26ed2fafd22ba3333a36383e5307b2d/biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/search-view.js#L79

Reply:
I refactored the code and I now use the pic-sure-dialog-view. This allowed me to remove both of the link-redirect-warning files and will make the code more robust. Thanks for calling it out.